javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/muzzle/collector/MuzzleCodeGenerator.java

```diff
@@ -110,10 +119,11 @@ public void visit(
     public FieldVisitor visitField(
         int access, String name, String descriptor, String signature, Object value) {
       if (MUZZLE_REF_MATCHER_FIELD_NAME.equals(name)) {
-        // muzzle field has been generated
-        // by previous compilation
-        // ignore and recompute in visitEnd
-        return null;
+        generateReferenceMatcherField = false;
+        log.warn(
```

Comment:
`warn` sounds like somebody has to do something about it. Is it the case here?

Reply:
True, `info` will be more appropriate